sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -19,9 +19,11 @@ export const LATEST_API_VERSION = "7.2";
  */
 export interface CertificateClientOptions extends coreHttp.PipelineOptions {
   /**
-   * The accepted versions of the KeyVault's service API.
+   * The Key Vault API version. By default requests will be sent to the latest API version.
```

Comment:
> By default requests will be sent to the latest API version.

What happens if KeyVault releases a version 8.0 that is backwards-incompatible? Is this a scenario we've planned for? (Do we generate a new major version of our package? New client in existing package?)

/cc @bterlson @xirzec

Reply:
I'm probably just phrasing it poorly. We would continue to make requests against the latest _known_ version (which will be 7.2 once this version GAs) by default.

I'd love suggestions on how I can phrase that better to avoid confusion though - so let me know if you have a recommendation!

Reply:
I am thinking in the more general case. Tagged Brian and Jeff because this is the first time I've seen multi-version support and wanted to know. :smile:

Reply:
We don't have a great story for supporting breaking changes between service versions that I know about. It's possible we could paper over the differences in convenience and maybe even have multiple generated clients underneath? It really depends on the nature of the breaks. 

If there was a sufficiently radical rewrite of a service, it might end up being best to cut a new major version ourselves.

Reply:
> maybe even have multiple generated clients underneath?

This is what python is doing. However, one concern is that we don't want to have significant increase in bundle size for browser.

Reply:
I think this concern is a bit beyond our current plans for Key Vault 🤔 what would be a viable action item for this release?